app/javascript/packs/document-capture.tsx

```diff
@@ -127,7 +98,7 @@ function trackEvent(event, payload) {
   const keepAlive = () =>
     window.fetch(keepAliveEndpoint, {
       method: 'POST',
-      headers: /** @type {string[][]} */ ([csrf && ['X-CSRF-Token', csrf]].filter(Boolean)),
+      headers: [csrf && ['X-CSRF-Token', csrf]].filter(Boolean) as [string, string][],
```

Comment:
I snuck in a TypeScript file conversion as part of the pull request, but this line was the only change needed to satisfy the new errors. See the original commit 935b91d to see the change more clearly.